lbrynet/core/client/BlobRequester.py

```diff
@@ -354,6 +354,10 @@ def _handle_availability(self, response_dict, request):
         log.debug("Received a response to the availability request")
         # save available blobs
         blob_hashes = response_dict['available_blobs']
+        if not blob_hashes:
+            # should not send any more requests as it doesnt have any blob we need
```

Comment:
What if it has other blobs for other streams? Won't this punish it globally for not having the blobs for this stream?

Reply:
@shyba ? Is this something we want to get into the next patch release?

Reply:
Yes, we just talked about it on the standup. The score seems local to the blob requester, since each file is a downloader and each downloader has its own requester. Jack will take a look on it and see if its ok to do this way or if there is a better approach.